hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/SchedulerException.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.mesos.hdfs.scheduler;
+
+/**
+ * Exceptions in the scheduler which likely result in the scheduler being shutdown.
+ */
+public class SchedulerException extends RuntimeException {
+  public SchedulerException() {
```

Comment:
I don't think we need an empty constructor here. The other constructors look good.